app/views/application/menus/_inquiries_list_link.html.erb

```diff
@@ -1,5 +1,5 @@
-<li class="<%= "active" if current_page?(inquiries_path) %>">
-  <%= link_to inquiries_path do %>
+<li class="<%= "active" if current_page?(organization_inquiries_path(organization_id: @organization.id)) %>">
+  <%= link_to organization_inquiries_path(organization_id: @organization.id) do %>
```

Comment:
Should be possible to directly use the shorter form `organization_inquiries_path(@organization)`.